spec.bs

```diff
@@ -1141,60 +1145,83 @@ element with arbitrary URLs *not* produced by config-generating APIs for develop
- * A <dfn for=fencedframeconfig>config</dfn>, a [=fenced frame config=]
+ * A <dfn for=fencedframeconfig>sharedStorageContext</dfn>, a [=string=]
+ * A <dfn for=fencedframeconfig>containerWidth</dfn>, a {{FencedFrameConfigSize}} or null
+ * A <dfn for=fencedframeconfig>containerHeight</dfn>, a {{FencedFrameConfigSize}} or null
+ * A <dfn for=fencedframeconfig>contentWidth</dfn>, a {{FencedFrameConfigSize}} or null
+ * A <dfn for=fencedframeconfig>contentHeight</dfn>, a {{FencedFrameConfigSize}} or null
 
 <div algorithm="containerWidth getter">
-  The {{FencedFrameConfig/containerWidth}} IDL attribute getter steps are:
-
-  1. If [=this=]'s [=fencedframeconfig/config=]'s [=fenced frame config/container size=] is null,
-     return null.
-
-  1. Return [=this=]'s [=fencedframeconfig/config=]'s [=fenced frame config/container size=]'s
-     [=size/width=].
+  The {{FencedFrameConfig/containerWidth}} IDL attribute getter steps are to return [=this=]'s
+  [=fencedframeconfig/containerWidth=].
 </div>
 
 <div algorithm="containerHeight getter">
-  The {{FencedFrameConfig/containerHeight}} IDL attribute getter steps are:
+  The {{FencedFrameConfig/containerHeight}} IDL attribute getter steps are to return [=this=]'s
+  [=fencedframeconfig/containerHeight=].
+</div>
 
-  1. If [=this=]'s [=fencedframeconfig/config=]'s [=fenced frame config/container size=] is null,
-     return null.
+<div algorithm="contentWidth getter">
+  The {{FencedFrameConfig/contentWidth}} IDL attribute getter steps are to return [=this=]'s
+  [=fencedframeconfig/contentWidth=].
+</div>
 
-  1. Return [=this=]'s [=fencedframeconfig/config=]'s [=fenced frame config/container size=]'s
-     [=size/height=].
+<div algorithm="contentHeight getter">
+  The {{FencedFrameConfig/contentHeight}} IDL attribute getter steps are to return [=this=]'s
+  [=fencedframeconfig/contentHeight=].
 </div>
 
-<div algorithm="contentWidth getter">
-  The {{FencedFrameConfig/contentWidth}} IDL attribute getter steps are:
+<div algorithm>
+  The <dfn method for=FencedFrameConfig>setSharedStorageContext(|contextString|)</dfn> method steps
+  are to set [=this=]'s [=fencedframeconfig/sharedStorageContext=] to |contextString|.
+</div>
 
-  1. If [=this=]'s [=fencedframeconfig/config=]'s [=fenced frame config/content size=] is null,
-     return null.
+<div algorithm="FencedFrameConfig serializer">
+  {{FencedFrameConfig}} objects are [=serializable objects=]. Their [=serialization steps=], given
+  |value|, |serialized|, and |forStorage| are:
 
-  1. If [=this=]'s [=fencedframeconfig/config=]'s [=fenced frame config/content size=]'s [=content
-     size/visibility=] is "<a for=visibility>`transparent`</a>", return the [=fenced frame
-     config/content size=]'s [=content size/value=]'s [=size/width=].
+  1. If |forStorage| is true, then throw a {{DataCloneError}} {{DOMException}}.
+  
+  1. Set |serialized|.`[[Urn]]` to the value of |value|'s [=fencedframeconfig/urn=] attribute.
```

Comment:
What you're accessing here with `[=fencedframeconfig/urn=]` isn't an IDL attribute, it is just a normal member. So you don't have to add `to the value of |value|'s foo attribute`. You can just have `to |value|'s [=fencedframeconfig/foo=]`. Here and elsewhere.